### PR TITLE
Named map auth tests

### PR DESCRIPTION
**Pull request by enekid**
Test that only *master* API Keys can access admin named map endpoints:
- list
- create
- get
- update
- delete

### CODE REVIEW

File: test/acceptance/auth/authorization.js [hunk 1]
```diff
@@ -354,110 +329,604 @@ describe('authorization', function() {
...
+
+            it('should fail getting a named map using a regular apikey token', function (done) {
+                const apikeyTokenCreate = 1234;
+                const apikeyTokenGet = 'regular1';
+
+                const template = Object.assign({}, templateBase, { name: templateBase.name + '-get-regular' });
+
+                const testClientCreate = new TestClient(template, apikeyTokenCreate);
+
+                testClientCreate.createTemplate({}, function (err, res, template) {
```
Comment (dgaubert) [inline test/acceptance/auth/authorization.js]: Really nice work isolating each test. A test should not depend on other tests. 💪

File: test/acceptance/auth/authorization.js [hunk 2]
```diff
@@ -354,110 +329,604 @@ describe('authorization', function() {
...
+        });
 
-        const template = {
-            version: '0.0.1',
-            name: 'auth-api-template-private',
-            placeholders: {
-                buffersize: {
-                    type: 'number',
-                    default: 0
+        describe('DELETE Named Map', function () {
```
Comment (dgaubert) [inline test/acceptance/auth/authorization.js]: I love the structure of these test. ;)

File: test/support/test-client.js [hunk 3]
```diff
@@ -1367,3 +1367,162 @@ TestClient.prototype.getNamedTile =  function (name, z, x, y, format, options, c
         });
     });
 };
+
+TestClient.prototype.createTemplate = function (params, callback) {
```
Comment (dgaubert) [inline test/support/test-client.js]: This is going to be pretty useful for further tests. 👍

**Comment by enekid** [comment]
Thanks for the nice words!

**Comment by oleurud** [comment]
:+1:

**Pull request merged**